### DISCUSSION TITLE
En Büyük Sayı

### DISCUSSION THREAD

**Discussion by AcelyaP** [sonsuzus/sonsuzus#328] [Soru Cevap]
D rakamlarına sahip olan N tamsayısı verildiğinde . N'nin içinden tam olarak K tamsayılarını silin.
Tam K rakamlarını sildikten sonra N'den oluşabilecek en büyük sayıyı bulun.

**Giriş Formatı:**
- İlk satır iki boşlukla ayrılmış tamsayı içerir N, K.

**Çıkış biçimi:**
N'den oluşturulabilen mümkün olan en büyük sayıyı yazdır tam olarak K çıkardıktan sonra rakamlardan.

**Kısıtlamalar:**
1≤N≤10¹⁸
1≤K≤3
K≤D

**Comment by sonsuzus** [comment]
> N tamsayısı verildiğinde D'ye sahip olan rakamlar. Tam olarak silmeniz gerekir K tamsayısındaki rakamlar N.
N'den oluşturulabilecek mümkün olan en büyük sayıyı bulun tam olarak K çıkardıktan sonra rakamlar.

Bu nasıl bir cümle :)) Düzenle lütfen soruyu.
